tests/test_type_coverage.py

```diff
@@ -54,7 +54,9 @@ def visit_FunctionDef(
             if arg.annotation:
                 self.assert_no_strings(arg.annotation)
                 typed_count += 1
-            elif index == 0 and self._classdefs and arg.arg in {"self", "cls"}:
+            elif index == 0 and self._classdefs and arg.arg in {"self",
+                                                                "cls",
+                                                                "mcls"}:
```

Comment:
Here I modified to include `mcls`, because some `@classmethod` are defined with first argument with such name. Since Flake8 doesn't complain about this, I thought logical to update this piece of code.